src/es-module-shims.js

```diff
@@ -295,7 +297,8 @@ function getOrCreateLoad (url, fetchOpts, source) {
       if (childFetchOpts.integrity)
         childFetchOpts = Object.assign({}, childFetchOpts, { integrity: undefined });
       return getOrCreateLoad(r, childFetchOpts).f;
-    }).filter(l => l));
+    }));
+    load.d = load.d.filter(l => l);
```

Comment:
Rather than reassigning, we can just run the filter on a bracket expression around the await.